impeller/aiks/picture.cc

```diff
@@ -3,11 +3,23 @@
 // found in the LICENSE file.
 
 #include "impeller/aiks/picture.h"
+#include <memory>
 
 #include "impeller/entity/entity.h"
+#include "impeller/renderer/render_target.h"
 
 namespace impeller {
 
-//
+std::shared_ptr<Image> Picture::RenderToImage(AiksContext& renderer) {
+  auto coverage = pass->GetElementsCoverage();
+  auto target = RenderTarget::CreateOffscreen(
+      *renderer.context_, ISize(coverage->GetRight(), coverage->GetBottom()));
+
+  if (!renderer.Render(*this, target)) {
+    return nullptr;
+  }
+
+  return std::make_shared<Image>(target.GetRenderTargetTexture());
```

Comment:
This is a GPU resident texture right?

The API exposed via toImage expects a CPU resident image.

Reply:
We can mark the new texture as host visible and have `Image` handle the shared buffer transfer internally. Would prefer to punt that down the road to when we add in stuff like `ReadPixels` (if/when we need it).

Reply:
This isn't exposed up through the Dart API AFAICT, right?

If so, let's just document the behavior in a comment here.

Reply:
Nah this is only Impeller internal right now. Added a comment about the visibility.